src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text_Format_Decimal.txt

```diff
@@ -0,0 +1,23 @@
+#SETUP: disable:NumberIsFloat
```

Comment:
>disable:NumberIsFloat [](http://example.com/codeflow?start=8&length=21)

why does decimal vs. float matter?

Reply:
removed this and created based on PowerFxV1 tag instead.